packages/components/addon/components/hds/dropdown/list-item/copy-item.hbs

```diff
@@ -8,5 +8,5 @@
       class="hds-dropdown-list-item__copy-item-title hds-typography-body-100 hds-font-weight-semibold"
     >{{@copyItemTitle}}</div>
   {{/if}}
-  <Hds::Copy::Snippet @textToCopy={{this.text}} @isTruncated={{@isTruncated}} />
+  <Hds::Copy::Snippet @color="secondary" @textToCopy={{this.text}} @isTruncated={{this.isTruncated}} />
```

Comment:
Just a note: one of the originally confusing things here was that design is using color in context, so our typical primary/secondary colors are not what they wanted to be used for this component. I'm pretty tired right now so it's entirely possible you have this correct, but since I found it confusing when originally implementing this component, I wanted to make sure to mention it.

Reply:
@MelSumner I'm not following, can you please clarify what you mean? Is there a question or concern that you have about how this has been updated?